File: 12/Net5.SignalR.Chat/Net5-SignalR-Chat-Client/src/app/models/room-user.ts
export class RoomUser {
  public RoomUserId?: number;
  public RoomId: number;
  public UserId: number;
  public Status: string;

  constructor() {
    this.RoomUserId = 0;
    this.RoomId = 0;
    this.UserId = 0;
    this.Status = '';
  }
}

File: 12/Net5.SignalR.Chat/Net5.SignalR.Chat.BackEnd/Infrastructure/Data/Entities/RoomUser.cs
namespace Net5.SignalR.Chat.BackEnd.Infrastructure.Data.Entities
{
    public class RoomUser
    {
        public int RoomUserId { get; set; }
        public int RoomId { get; set; }
        public int UserId { get; set; }
        public string Status { get; set; }

        public virtual Room Room { get; set; }
        public virtual User User { get; set; }
    }
}